packages/webview/src/component/terminal/terminal-colors.ts

```diff
@@ -32,8 +32,52 @@ export const ansi256Colours = [
   '\u001b[34;1m', // bright blue
 ];
 
+// ANSI colors for log levels
+const LOG_LEVEL_COLORS: Record<string, string> = {
+    DBG: '\u001b[36m', // cyan
+    DEBUG: '\u001b[36m',
+    INF: '\u001b[32m', // green
+    INFO: '\u001b[32m',
+    WRN: '\u001b[33m', // yellow
+    WARN: '\u001b[33m',
+    WARNING: '\u001b[33m',
+    ERR: '\u001b[31m', // red
+    ERROR: '\u001b[31m',
+    FATAL: '\u001b[31;1m', // bright red
+    TRACE: '\u001b[35m', // magenta
+};
```

Comment:
Are these the colors used, or are we loading them from elsewhere?

Reply:
These are the ANSI colors supported by the terminal. If we want to change the color to be rendered at the end, we may want to change them in the theme of the terminal. I think it could be part of another PR